hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOMRatisSnapshots.java

```diff
@@ -26,7 +26,7 @@
 import org.apache.commons.lang3.RandomStringUtils;
 import org.apache.hadoop.hdds.conf.OzoneConfiguration;
 import org.apache.hadoop.hdds.utils.db.DBCheckpoint;
-import org.apache.hadoop.ozone.MiniOzoneCluster;
+import org.apache.hadoop.ozone.MinOzoneCluster;
```

Comment:
This looks like a typo, causes compile errors.

```
TestOMRatisSnapshots.java:[29,31] cannot find symbol
Error:    symbol:   class MinOzoneCluster
Error:    location: package org.apache.hadoop.ozone
```

```suggestion
import org.apache.hadoop.ozone.MiniOzoneCluster;
```

Reply:
Thanks @adoroszlai.  Commit correction.  It seems to slipped through.  Verified on local build.  Should be ok.